public/utils/helpers.js

```diff
@@ -75,7 +75,7 @@ export const getDigitId = (length = 6) =>
 
 // Assumes that values is an array of objects with "name" inside
 export const getUniqueName = (values, prefix) => {
-  const lastValue = values.at(-1);
+  const lastValue = _.last(values);
```

Comment:
For context, the following test started failing when executing `yarn run test:jest`. Refactored the [.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) call to [_.last()](https://lodash.com/docs/4.17.15#last). 
```
 FAIL  public/components/FeatureAnywhereContextMenu/AddAlertingMonitor/CreateNew/CreateNew.test.js (24.469 s)
  ● CreateNew › renders

    TypeError: values.at is not a function

      78 |   const lastValue =
      79 |     // _.get(values, values.length - 2)
    > 80 |     values.at(-1);
         |            ^
      81 |   const lastName = lastValue ? lastValue.name : '';
      82 |   const lastDigit = Number.parseInt(lastName.match(/\d+$/)?.[0] || 0, 10);
      83 |

      at getUniqueName (public/utils/helpers.js:80:12)
      at getInitialTriggerValues (public/pages/CreateTrigger/components/AddTriggerButton/utils.js:30:39)
      at getInitialValues (public/pages/CreateMonitor/containers/CreateMonitor/utils/helpers.js:49:51)
      at Object.<anonymous> (public/components/FeatureAnywhereContextMenu/AddAlertingMonitor/CreateNew/CreateNew.test.js:32:42)